packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -270,6 +282,24 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
     );
   }
 
+  Widget _buildStatusLine(BuildContext context) {
+    final inset = DevToolsScaffold.borderInsets.bottom;
+
+    return Container(
+      height: 48.0,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: <Widget>[
```

Comment:
remove `<Widget>`

Reply:
done

Reply:
(looks like we have ~27 other places w/ this pattern)